WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllViewModel.kt

```diff
@@ -247,3 +248,53 @@ class YearlyViewAllPostsAndPagesViewModel
         R.string.stats_view_top_posts_and_pages
 )
 // endregion
+
+// region ViewAllReferrersViewModel
+class DailyViewAllReferrersViewModel
```

Comment:
What do you think about merging all these view models and passing the use case type as a parameter to the `ViewModel`? The `ViewModel` can inject a list of use cases and use the one that matches the use case type. I think it would reduce the boilerplate